pages/common/transcrypt.md

```diff
@@ -0,0 +1,24 @@
+# transcrypt
+
+> Transparently encrypt files within a git repository.
+> More information: <https://github.com/elasticdog/transcrypt>.
+
+- Initialize an unconfigured repository:
+
+`transcrypt`
+
+- List the currently encrypted files:
+
+`git ls-crypt`
+
+- Display the credentials of a configured repository:
+
+`transcrypt --display`
+
+- Initialize and decrypt a fresh clone of a configured repository:
+
+`transcrypt -c <cipher> -p '<password>'`
```

Comment:
Are there any long forms of these `-c` and `-p` arguments? We prefer long-form arguments in tldr pages.

Also, is there a way to get it to prompt for a password or use a password file instead? Providing the password directly on the command line is rather insecure, since it will be stored in the shell history.

Reply:
@sbrl 
- Yes, there are long forms for those :) 
- And also yes: users can simply use `transcrypt` to enter the cipher and the password via the prompt (it is the exact same command as the one to initially setup transcrypt actually! ).

Should I remove this command from the page, and better explain that the first one can also be used to decrypt a repo ?

Thank you for your review !

Reply:
- Cool, If we could change the examples to use the long forms of the arguments instead, that'd be great!
 - Right. Perhaps we could keep specifying the cipher here and just remove the password bit?

It seems odd that it requires you to explicitly specify the cipher used for encryption. I'd have thought that it would store that information somewhere.

Many thanks for the new page!

Reply:
I agree with removing the password part if it can be done with prompt instead.

Reply:
@sbrl Changes applied !
> It seems odd that it requires you to explicitly specify the cipher used for encryption. I'd have thought that it would store that information somewhere.

True... It seems like it is considered as part of the "credentials"